velox/dwio/common/InputStream.h

```diff
@@ -231,9 +216,8 @@ class ReferenceableInputStream : public InputStream {
  public:
   explicit ReferenceableInputStream(
       const std::string& /* UNUSED*/,
-      const MetricsLogPtr& metricsLog = MetricsLog::voidLog(),
-      IoStatistics* FOLLY_NULLABLE stats = nullptr)
```

Comment:
You cannot remove it for now because it is sometime used without `BufferedInput` outside Velox